server-apollo/index.js

```diff
@@ -162,6 +181,10 @@ Mutation: {
 }
 }
 
+const conflictDetected = (server, client) => {
```

Comment:
ping @darahaye
We can have something like this method when conflict is detected.
Server side is just mock to showcase conflict resolution based on inmem data. We can effectively now showcase that conflict happens on the client and continue writing some handlers and other helpers that are needed.